erpnext/manufacturing/doctype/bom_update_tool/bom_update_tool.js

```diff
@@ -22,6 +22,24 @@ frappe.ui.form.on('BOM Update Tool', {
 		frm.disable_save();
 	},
 
+	replace: function(frm) {
+		if (frm.doc.current_bom && frm.doc.new_bom) {
+			frappe.call({
+				method: "erpnext.manufacturing.doctype.bom_update_tool.bom_update_tool.enqueue_replace_bom",
+				freeze: true,
+				args: {
+					args: {
+						"current_bom": frm.doc.current_bom,
+						"new_bom": frm.doc.new_bom
+					}
+				},
+				callback: function() {
+					frappe.msgprint(__("BOM replaced successfully"));
```

Comment:
As we have enqueued the task, this message is wrong. Actually the message is not needed, you have added a msgprint in server side.